sgv2-service-common/src/main/java/io/stargate/sgv2/common/cql/builder/Literal.java

```diff
@@ -15,15 +15,20 @@
  */
 package io.stargate.sgv2.common.cql.builder;
 
-class Literal implements Term {
+import io.stargate.bridge.proto.QueryOuterClass.Value;
+import java.util.Objects;
 
-  private final Object value;
+public class Literal implements Term {
```

Comment:
mentioned already in #1796 , this is a nice candidate for immutable..

Reply:
Maybe we can address this in a follow-up ticket, I don't want to delay this...

But I think we are over-using `@Value.Immutable`: in this case the class is already immutable and it doesn't need value equality. So it wouldn't add any value.

Reply:
And just after writing this, I notice that I've implemented `equals` and `hashCode` on another branch... But still, it's not that hard to do manually.